package.json

```diff
@@ -119,4 +119,4 @@
     "ts-loader": "5.2.2",
     "typescript": "3.1.3"
   }
-}
+}
```

Comment:
?

Reply:
hmm i don't know why my linter would have done that 🤔